### ISSUE TITLE
Sound gone again on 2.1.6?

### ISSUE THREAD

**Issue by jonrandy** [mmartinortiz/retro-virtual-machine-snap#2]
The sound appears to be broken again?

**Comment by mmartinortiz** [comment]
I just checked and the sound works in my computer for version 2.1.6. Using the Latest/Edge channel.

Have you checked that the `alsa` and `audio-playback` permissions are on?

**Comment by jonrandy** [comment]
![permissions](https://user-images.githubusercontent.com/1510194/227693283-086a1a08-d41f-4c06-8bc2-71a461cc380d.png)
I haven't changed anything

**Comment by mmartinortiz** [comment]
I have tested it installing the snap into a new Mate VM. 

At the first run, it had no sound. Then I checked the permissions and Alsa was disabled. After enabling it, the sound was working.

![Screenshot_ubuntu-mate22 04_2023-03-25_09:40:57](https://user-images.githubusercontent.com/4763770/227707086-216839ea-935f-4c9a-ae4b-2cd759d9b660.png)

I'm using the `preview-edge` version of the Snap Store. I do not know if that makes any difference for enabling interfaces

![image](https://user-images.githubusercontent.com/4763770/227707128-62b03fbb-cd9b-4e85-ba47-4449782aae71.png)

But it is quite strange that, without changing anything, you lost the sound. Does it happen with all the emulated systems? Reinstalling is another option, but make sure to save the state of your machines if you have games on going.
